scripts/build/toolchain/termux_setup_toolchain_27c.sh

```diff
@@ -3,10 +3,10 @@ termux_setup_toolchain_27c() {
 	export CPPFLAGS=""
 	export LDFLAGS="-L${TERMUX_PREFIX}/lib"
 
-	export AS=$TERMUX_HOST_PLATFORM-clang
-	export CC=$TERMUX_HOST_PLATFORM-clang
-	export CPP=$TERMUX_HOST_PLATFORM-cpp
-	export CXX=$TERMUX_HOST_PLATFORM-clang++
+	export AS="ccache-$TERMUX_HOST_PLATFORM-clang"
```

Comment:
Supporting ability to disable ccache based on a conifg set in .termuxrc would be nice. ccache doesnt always work really well with all builds